### PR TITLE
Add Vocalizer Expressive Driver 3.1.8

### PR DESCRIPTION
**Pull request by nvdaes**
### Release information

- Name: Vocalizer Expressive 1.1 Driver
- Author: Tiflotecnia, LDA.
- Repo: https://github.com/ruifontes
- Version: 3.1.8
- Update channel: stable
- NVDA compatibility: 2022.1 and beyond

### Changelog

- Changing manifest.ini to be compatible with NVDA 2023.1;
- Introducing automatic update system.

### Additional information

[Announcement on the add-ons mailing list](https://nvda-addons.groups.io/g/nvda-addons/message/20260)

cc: @ruifontes

### CODE REVIEW

File: get.php
Comment (josephsl) [inline get.php:158]: Aren't there drivers for Vocalizer classic (version 2/direct) as well? If yes, may I suggest changing the key a bit? If not and if Vocalizer Direct (version 2) will not be updated or no updates to be provided, then let's keep the key defined.

**Comment by ruifontes** [comment]
Vocalizer Automotive 2.1.4 is in preparation...
Should be released untill monday...

Best regards, 

Rui Fontes
Tiflotecnia, Lda.

**Comment by josephsl** [comment]
Hi, I see- in that case, I propose either holding off on this PR until Monday or separate Vocalizer 2 and 3 keys. Thanks.

**Comment by nvdaes** [comment]
@josephsl, I think it"s better that you review this and approve if appropriate. Unique key ids are required, but we don"t know if addons with similar names will be send here some day. If vocalizer automotive is sent, we can use a different key id. In fact I have though about using another key id before creating this PR, but finally I have use vocalizer thinking that the summary of this add-on contain the number 2 and I don"t know if numbers are supported in key ids and, since vocalizer is a long name, finally I used it like that. We deppend on others so that this is merged and I think that maybe confusing to wait to see if other add-ons are sent. We can use then vocalizerauto or something.Enviado desde mi iPhoneEl 18 feb 2023, a las 22:38, Joseph Lee ***@***.***> escribió:﻿
Hi, I see- in that case, I propose either holding off on this PR until Monday or separate Vocalizer 2 and 3 keys. Thanks.

**Comment by XLTechie** [comment]
@nvdaes Numbers are supported in key IDs. See, for example, "access8math", 
"mp3dc", or "nvda3208".

**Comment by josephsl** [comment]
Hi, dictionary keys are strings, so numbers can be used (after all, they are strings as far as keys are concerned). I think it would be best to call them “vocalizerautomotive” and “vocalizerexpressive”, although this can cause issues for translators (perhaps we could call them “vocautomotive” and “vocexpressive”). Thanks.

**Comment by nvdaes** [comment]
yes, also Luke. I know access8math and nvda3208. For any reason, seeing the number in the add-on summary, I have thought that thit may cause difficulties in the key id and I have avoided it. I"ll change it now adding the expressive word, that maybe clearer.Enviado desde mi iPhoneEl 18 feb 2023, a las 23:41, Joseph Lee ***@***.***> escribió:﻿
Hi, dictionary keys are strings, so numbers can be used (after all, they are strings as far as keys are concerned). I think it would be best to call them “vocalizerautomotive” and “vocalizerexpressive”, although this can cause issues for translators (perhaps we could call them “vocautomotive” and “vocexpressive”). Thanks.

**Review by josephsl — APPROVED** [review]

**Pull request merged**
